src/Symfony/Component/Console/Command/Command.php

```diff
@@ -284,6 +284,10 @@ public function setCode($code)
             throw new \InvalidArgumentException('Invalid callable provided to Command::setCode.');
         }
 
+        if (PHP_VERSION_ID >= 50400 && $code instanceof \Closure) {
```

Comment:
The Closure class exists since 5.3, but Closure::bind has been introduced in 5.4, so the check is required :)